gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -45,6 +51,17 @@ class GazeboRosWheelSlipPrivate
   // Event handler to set slip compliance values for individual wheels based
   std::shared_ptr<rclcpp::ParameterEventHandler> parameter_event_handler_;
   rclcpp::ParameterEventCallbackHandle::SharedPtr parameter_set_event_callback_;
+  // Publish wheel slip
+  rclcpp::Publisher<gazebo_msgs::msg::WheelSlip>::SharedPtr slip_publisher_;
+
+  /// Period in seconds
+  double update_period_;
+
+  /// Keep last time an update was published
+  gazebo::common::Time last_update_time_;
+
+  /// Pointer to the update event connection.
+  gazebo::event::ConnectionPtr update_connection_;
```

Comment:
please add `publisher_` as a prefix to `update` in these 3 new variable names to match the changes in 53c792ba286876e9e477153ee02cdb5f8fa825e6

Reply:
Fixed in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1331/commits/be39f37ef3703a961f54c66110e75315f5d3b825.